test/simple_module_test.dart

```diff
@@ -0,0 +1,27 @@
+@TestOn('vm || browser')
+import 'package:test/test.dart';
+import 'package:w_module/w_module.dart';
```

Comment:
This is another spot where you can import just `simple_module.dart` rather than all of w_module.

```dart
import 'package:w_module/src/simple_module.dart';
```